lib/dstu2_resources.rb

```diff
@@ -361,104 +361,270 @@ module Resources
+              "id": "M22335052",
+              "meta": {
+                "versionId": "1",
+                "lastUpdated": "2016-05-03"
+              },
+              "text": {
+                "status": "generated",
+                "div": "&lt;div>&lt;p>&lt;b>Immunization&lt;/b>&lt;/p>&lt;p>&lt;b>Vaccine&lt;/b>: diphtheria/pertussis, acel/tetanus ped&lt;/p>&lt;p>&lt;b>Date Attempted&lt;/b>: 2016-05-02&lt;/p>&lt;p>&lt;b>Given&lt;/b>: No&lt;/p>&lt;p>&lt;b>Administered by&lt;/b>: Dr. Who&lt;/p>&lt;p>&lt;b>Reasons Not Given&lt;/b>: Allergy to Neomycin (Disorder)&lt;/p>&lt;/div>"
+              },
+              "status": "stopped",
+              "date": "2016-05-02",
+              "vaccineCode": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/sid/cvx",
+                    "code": "20",
+                    "display": "DTaP",
+                    "userSelected": false
+                  }
+                ],
+                "text": "diphtheria/pertussis, acel/tetanus ped"
+              },
+              "patient": {
+                "reference": "Patient/2100031",
+                "display": "River Song"
+              },
+              "wasNotGiven": true,
+              "reported": false,
+              "performer": {
+                "reference": "Practitioner/7750958",
+                "display": "Dr. Who"
+              },
+              "location": {
+                "reference": "Location/4048128",
+                "display": "Baseline East"
+              },
+              "explanation": {
+                "reasonNotGiven": [
+                  {
+                    "coding": [
+                      {
+                        "system": "http://hl7.org/fhir/v3/ActReason",
+                        "code": "MEDPREC",
+                        "display": "medical precaution",
+                        "userSelected": false
+                      }
+                    ],
+                    "text": "Allergy to Neomycin (Disorder)"
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "fullUrl": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Immunization/HM4234847",
+            "resource": {
+              "resourceType": "Immunization",
+              "id": "HM4234847",
+              "meta": {
+                "versionId": "0",
+                "lastUpdated": "2016-05-13T12:03:36-0500"
+              },
+              "text": {
+                "status": "generated",
+                "div": "&lt;div>&lt;p>&lt;b>Immunization&lt;/b>&lt;/p>&lt;p>&lt;b>Vaccine&lt;/b>: Hepatitis B&lt;/p>&lt;p>&lt;b>Date Attempted&lt;/b>: 2016-05-13T12:03:36-0500&lt;/p>&lt;p>&lt;b>Given&lt;/b>: No&lt;/p>&lt;p>&lt;b>Administered by&lt;/b>: Dr. Who&lt;/p>&lt;p>&lt;b>Reasons Not Given&lt;/b>: Patient Refuses&lt;/p>&lt;/div>"
+              },
+              "status": "on-hold",
```

Comment:
We are not currently honor the Argonaut value set. There is a JIRA logged to switch to it.